gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -702,6 +719,181 @@ public void testConfigureBucketsWithRootBucketButNoSystemBucket() throws IOExcep
     assertThat(fs.initUri).isEqualTo(initUri);
   }
 
+  @Test
+  public void create_storage_statistics() throws IOException {
+    GoogleHadoopFileSystem myGhfs = createInMemoryGoogleHadoopFileSystem();
+    URI fileUri = GoogleCloudStorageFileSystemIntegrationTest.getTempFilePath();
+    Path filePath = ghfsHelper.castAsHadoopPath(fileUri);
+    GhfsStorageStatistics StorageStats = myGhfs.getStorageStatistics();
+
+    try (FSDataOutputStream fout = myGhfs.create(filePath, true, 1, (short) 1, 1)) {
+      fout.writeBytes("Test Content");
+      fout.close();
+    }
+    assertThat(StorageStats.isTracked("op_create")).isTrue();
```

Comment:
Storagestats also implements `getLong(String key)` . Could you please assert this method as well. To ensure correctness of the values ?  ** for al tests **

Reply:
Updated with tests for getLong(String key)